Nos.xcodeproj/project.pbxproj

```diff
@@ -2198,15 +2186,18 @@
 		C9DEBFD5298941000078B43A /* Nos.xcdatamodeld */ = {
 			isa = XCVersionGroup;
 			children = (
-				5B503F632A291D480098805A /* Nos 7.xcdatamodel */,
+				5BFF66AF2A4B55FC00AA79DD /* Nos 10.xcdatamodel */,
+				5BEC18CA2A4B438700E6842E /* Nos 9.xcdatamodel */,
+				C9EDC9792A3A27D1009148D1 /* Nos 8.xcdatamodel */,
+				C9CF23152A378E8F00EBEC31 /* Nos 7.xcdatamodel */,
 				C91FFD392A09444D00743037 /* Nos 6.xcdatamodel */,
 				C9B678E529EF1BF200303F33 /* Nos 5.xcdatamodel */,
 				C9A0DAF329C870D500466635 /* Nos 4.xcdatamodel */,
 				CD4D210D29C350C9000F8188 /* Nos 3.xcdatamodel */,
 				C92DF80229C22E4F00400561 /* Nos 2.xcdatamodel */,
 				C9DEBFD6298941000078B43A /* Nos.xcdatamodel */,
 			);
-			currentVersion = 5B503F632A291D480098805A /* Nos 7.xcdatamodel */;
+			currentVersion = 5BFF66AF2A4B55FC00AA79DD /* Nos 10.xcdatamodel */;
```

Comment:
Do we need Nos 9.xcdatamodel? I think `fix-spinners` added Nos 8, so this PR should add Nos 9? Am I missing something?

Reply:
No, we don't need it. fix-spinners added 7 and 8 as well, so I though it was ok to maintain intermediate models. The number 9 was the datamodel as I developed it before collapsing RelayMetadata properties to Relay directly, maybe we can delete 9 and rename 10 later.

Reply:
I removed previous versions